components/brex/actions/list-transactions-for-primary-card-account/common.mjs

```diff
@@ -0,0 +1,42 @@
+import { axios } from "@pipedream/platform";
+
+export default {
+  async run ({ $ }) {
+    const {
+      max,
+      postedAtStart,
+    } = this;
+
+    let items = [];
+    let cursor;
+
+    const DEFAULT_LIMIT = 100;
+    const limit = max
+      ? Math.min(DEFAULT_LIMIT, parseInt(max))
+      : DEFAULT_LIMIT;
+
+    do {
+      const res = await axios($, this.brexApp._getAxiosParams({
+        method: "GET",
+        path: "/v2/transactions/card/primary",
+        params: {
+          limit,
+          posted_at_start: postedAtStart,
+          cursor,
+        },
+      }));
+
+      if (res.items) {
+        items = [
+          ...items,
+          ...res.items,
+        ];
+      }
+
+      cursor = res.next_cursor;
+    } while (items.length < max && cursor);
+
+    $.export("$summary", "Transactions successfully fetched");
```

Comment:
same comment here -- 
can you add a count of how many transactions we fetched?
"Successfully fetched 35 transactions" for example. Right now if we fetch 0 transactions, it's slightly confusing since their API just returns 0 results:
![image](https://user-images.githubusercontent.com/20780753/154127472-253f810c-7c05-40a1-8d7d-8fd372859014.png)

Reply:
Okay :)